radis/lbl/broadening.py

```diff
@@ -1876,11 +1884,20 @@ def _apply_lineshape_DLM(
         self.profiler.start("DLM_Distribute_lines", 3)
         # ... Initialize array on which to distribute the lineshapes
         if broadening_method in ["voigt", "convolve"]:
-            DLM = np.zeros((len(wavenumber_calc) + 2, len(wG), len(wL)))
-            # +2 to allocate one empty grid point on each side : case where a line is on the boundary
-            ki0 += 1
-            ki1 += 1
+            if radis.config["SPARSE_WAVERANGE"]:
+                # DLM is constructed in a sparse-way later
+                pass
+            else:
+                DLM = np.zeros((len(wavenumber_calc) + 2, len(wG), len(wL)))
```

Comment:
This was added in https://github.com/radis/radis/pull/367 to fix index issues with the former DLM version, I think it's one point on the left and one on the right    [ you don't need them in FFT because you're extending the range by *2 ]

Reply:
Right, in the FFT case you only add 'Nlines' points to the right, nothing to the left.
It's a minor point of course, but if not adding the point on the left fails, it could be indicative of deeper issues.